chingu-voyages/v51-tier3-team-33
# Expenses UI

# Description
Expenses and expense form UI layout

## Related to the following Jira/Trello issue 

- These issues pertain to Sprint 3

## Main changes explained:
Expenses UI layout include:
- Expenses page where users can create a new expense. There are breadcrumbs for testing purposes that can also be great for UX if we decide to keep them.
- Expense form includes the "General" and "Splitting" section
- Expense view includes "Edit" button
- Expense create includes "Create" and "Cancel" button
- Expense edit includes "Save" and "Cancel" button

## How to test:
1. check into current branch
2. do `pnpm install` and `pnpm run dev` to run this PR locally
3. go to `include url to view your component/feature` 
4. verify all links work
5. verify UI is clear for a seamless UX

## Screenshots or videos of changes:
![expenses](https://github.com/user-attachments/assets/b35d6a4f-cbc4-490d-aed6-90ec7a0f2397)
![view](https://github.com/user-attachments/assets/db4637d0-9b64-4a94-9ec1-2fb4c697188d)
![new](https://github.com/user-attachments/assets/5ebc4ba6-3938-4d5e-acf6-274f2840aa5d)
![edit](https://github.com/user-attachments/assets/33489e5c-7fda-41f1-8bc9-36575aa38999)
![general-section](https://github.com/user-attachments/assets/4a40c084-1ad1-4240-b85b-209fd4eb622a)
![splitting-section](https://github.com/user-attachments/assets/306f1c00-039e-496b-94e5-66730e7af1ce)

## Note:
Defining the UI will help us integrate the appropriate functionalities
